<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComputerDialog</class>
 <widget class="QDialog" name="ComputerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="button_info_computer">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>470</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add_new_scientist">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add new</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="computer_table">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>731</width>
     <height>381</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>42</number>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Year Built</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Development</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="search_box_computer">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>0</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>Search....</string>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_add_computer">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>430</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dropdown_what_to_look_at">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>3</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
